dbxio/core/cloud/__init__.py

```diff
@@ -0,0 +1,21 @@
+import enum
+
+from dbxio.core.cloud.aws import AWS_ClOUD_PROVIDER_NAME
+from dbxio.core.cloud.azure import AZURE_ClOUD_PROVIDER_NAME
+from dbxio.core.cloud.client.object_storage import ObjectStorageClient
+from dbxio.core.cloud.gcp import GCP_ClOUD_PROVIDER_NAME
+from dbxio.core.cloud.nebius_over_azure import NEBIUS_OVER_AZURE_ClOUD_PROVIDER_NAME
+
+
+@enum.unique
+class CloudProvider(str, enum.Enum):
+    AWS = AWS_ClOUD_PROVIDER_NAME
+    AZURE = AZURE_ClOUD_PROVIDER_NAME
+    GCP = GCP_ClOUD_PROVIDER_NAME
+    NEBIUS_OVER_AZURE = NEBIUS_OVER_AZURE_ClOUD_PROVIDER_NAME
```

Comment:
What is nebius_over_azure? Why not just nebius?

Reply:
Because Nebius has it's own authorization system, but we use here azure on nebius machines

Reply:
Removed Nebius as cloud provider